download.md

```diff
@@ -26,19 +26,19 @@ See: Official Pup build recipes at [Woof-CE on GITHUB]
 {: .table .table-striped .table-bordered }
 
 [Woof-CE on GITHUB]: https://github.com/puppylinux-woof-CE/woof-CE/tree/testing/woof-distro
-[sla]: http://slacko.eezy.xyz
```

Comment:
I haven't got https on that domain. It's a non login site. I can change links on that site though to https.

Reply:
@01micko 
Changes made as requested + oldforum links corrected